### ISSUE TITLE
[Bug]: NoClassDefFoundError BladeInjectionInfo

### ISSUE THREAD

**Issue by seanmtaylor** [laravel-idea/plugin#727]
### Bug description

When I load PhpStorm (`2022.3.2`) I get an IDE Internal Error which seems to prevent any of the blade highlighting and autocompletion from working.

### Plugin version

6.3.0.223

### Operating system

Linux - Ubuntu 22.04

### Relevant log output

```shell
java.lang.NoClassDefFoundError: com/jetbrains/php/blade/injection/BladeInjectionInfo
	at com.laravel_idea.plugin.extension.customConfig.models.blade.BladeDirectiveModel.getDirectives(BladeDirectiveModel.kt:14)
	at com.laravel_idea.plugin.extension.customConfig.models.blade.BladeModel.registerDirectives(BladeModel.kt:24)
	at com.laravel_idea.plugin.extension.customConfig.models.blade.BladeModel.register(BladeModel.kt:19)
	at com.laravel_idea.plugin.extension.customConfig.models.ConfigModel.register(ConfigModel.kt:35)
	at com.laravel_idea.plugin.extension.customConfig.LaravelCustomConfig.a(LaravelCustomConfig.kt:84)
	at com.laravel_idea.plugin.extension.customConfig.LaravelCustomConfig.updateWithoutIndices(LaravelCustomConfig.kt:17)
	at com.laravel_idea.plugin.extension.settings.LaravelIdeaSettings.afterProjectOpen(LaravelIdeaSettings.kt:289)
	at com.laravel_idea.plugin.extension.settings.b.runActivity(b.java:9)
	at com.intellij.ide.startup.impl.StartupManagerImpl.runActivityAndMeasureDuration(StartupManagerImpl.kt:340)
	at com.intellij.ide.startup.impl.StartupManagerImpl.access$runActivityAndMeasureDuration(StartupManagerImpl.kt:66)
	at com.intellij.ide.startup.impl.StartupManagerImpl$runPostStartupActivities$2$2$1.invoke(StartupManagerImpl.kt:279)
	at com.intellij.ide.startup.impl.StartupManagerImpl$runPostStartupActivities$2$2$1.invoke(StartupManagerImpl.kt:278)
	at com.intellij.openapi.progress.CancellationKt.withCurrentJob$lambda$0(cancellation.kt:17)
	at com.intellij.openapi.progress.Cancellation.withCurrentJob(Cancellation.java:60)
	at com.intellij.openapi.progress.CancellationKt.withCurrentJob(cancellation.kt:17)
	at com.intellij.openapi.progress.CoroutinesKt.blockingContext(coroutines.kt:142)
	at com.intellij.ide.startup.impl.StartupManagerImpl.runPostStartupActivities(StartupManagerImpl.kt:278)
	at com.intellij.ide.startup.impl.StartupManagerImpl.access$runPostStartupActivities(StartupManagerImpl.kt:66)
	at com.intellij.ide.startup.impl.StartupManagerImpl$runStartupActivities$2.invokeSuspend(StartupManagerImpl.kt:184)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:570)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:750)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:677)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:664)
Caused by: java.lang.ClassNotFoundException: com.jetbrains.php.blade.injection.BladeInjectionInfo PluginClassLoader(plugin=PluginDescriptor(name=Laravel Idea, id=com.laravel_idea.plugin, descriptorPath=plugin.xml, path=~/.local/share/JetBrains/Toolbox/apps/PhpStorm/ch-0/223.8617.59.plugins/Laravel Idea, version=6.3.0.223, package=null, isBundled=false), packagePrefix=null, instanceId=128, state=active)
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:217)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
	... 25 more
```

**Comment by adelf** [comment]
Hi, Sean.

Thanks for posting. It seems the Blade plugin isn't installed on your PhpStorm. I'll fix this exception in the next version.

**Comment by seanmtaylor** [comment]
Yep, that was it :facepalm: 

Thanks for the quick reply @adelf, enabling the bundled Blade plugin has everything working as expected again.

**Comment by adelf** [comment]
Fixed in the 6.3.1. Thanks
